templates/Config/config.yaml

```diff
@@ -0,0 +1,118 @@
+# From https://github.com/org-formation/org-formation-reference/blob/master/src/templates/080-aws-config-inventory/config.yml
+AWSTemplateFormatVersion: '2010-09-09'
+
+Metadata:
+  cfn-lint:
+    config:
+      ignore_checks: [W2001]
```

Comment:
Can you include a comment about what these checks are ignoring?

Reply:
added a general comment for these ignore above this section.